src/lib/concurrency/logging/group_commit_logger.cpp

```diff
@@ -0,0 +1,275 @@
+/*
+ *  The GroupCommitLogger gathers log entries in its buffer and flushes them to disk in a binary format:
```

Comment:
Could the logger type (directly flushing, group commit, ...) be independent of the log layout (text, binary)? Especially since it looks like that on the recovery side.